app/views/tools/index.html.haml

```diff
@@ -2,8 +2,19 @@
 
 - content_for :header, @title
 
-- if @articles.present?
-  - @articles.each do |article|
+- featured_articles = @articles.select(&:featured?)
+- if featured_articles.present?
+  .mb-3= render 'title', { title: 'Featured Tools' }
+  .mb-3.featured-articles.d-flex.flex-column.flex-md-row.justify-content-around.align-items-center.align-items-md-stretch
+    - featured_articles.each do |article|
+      = link_to blog_post_path(article.url_hash), class: "featured-article d-flex align-items-end" do
+        .featured-article-background-image{ style: "background-image: url(#{blog_cover_url(article)});" }
+        .featured-article-title= article.title
+  .mb-3= render 'title', { title: 'Other Tools' }
```

Comment:
These can be simplified: `= render 'title', { title: 'Other Tools' }` to `= render "title", title: "Other Tools"`, no need for the extra curly braces. Also, can you use double quotes for Ruby, as specified in the rubocop.yml file? For this project, double quotes are used for all strings regardless of interpolation or content.